src/version.c

```diff
@@ -2,8 +2,15 @@
 #include <string.h>
 #include "fmgr.h"
 
+#ifndef EXT_GIT_COMMIT
```

Comment:
Can we make this actually work? i.e. make the right commit appear in here. We'll probably need this and a #define for the extension version too. Also need to only set "windows" if _WIN32.

Reply:
I guess we could but it's really a dev thing used when testing non-release versions. As none of our current developers develop on Windows predominantly, I just put in this hack.

Reply:
I think we should fix this for release builds as well as windows builds but until then I would prefer to set it to `unknown` or something similar.

Reply:
Removed and now defined via Visual Studio compiler's command line